examples/chat/src/main.rs

```diff
@@ -69,6 +69,8 @@ use std::str::FromStr;
 use std::sync::{Arc, Mutex};
 use tracing::info;
 
+const APPLICATION_NAMESPACE: &[u8] = b"commonware-chat";
```

Comment:
Just because `chat`/`vrf` are examples, I recommend adding a short description for what an "APPLICATION_NAMESPACE" is.